examples/custom_type_derives.rs

```diff
@@ -16,7 +16,7 @@
 
 #[subxt::subxt(
     runtime_metadata_path = "examples/polkadot_metadata.scale",
-    generated_type_derives = "Clone, Debug"
+    generated_type_derives = "Clone"
```

Comment:
A comment here and on `runtime_metadata_path` would be good.

Reply:
Looking further, I think this whole example could use some attention. I am not sure what it actually does. :/
(Not a concern for this PR ofc)

Reply:
I gave the exampel a wee tidy; it's purpose appears solely to be showing off the `generated_type_derives` usage, so I made that a little clearer (and I didn't see a need for the strange way of calling `clone`.